src/test/java/pl/piekoszek/gorskimatches/controllers/ChallengeControllerTest.java

```diff
@@ -27,9 +27,11 @@ public class ChallengeControllerTest {
     private TokenService tokenService;
 
     @Test
-    void shouldReturnUuid() throws Exception {
-        mockMvc.perform(get("/api/challenge/generate"))
-                .andExpect(status().isOk());
+    void shouldCreateChallenge() throws Exception {
+        var uuid = mockMvc.perform(get("/api/challenge/generate"))
+                .andExpect(jsonPath("$").isString())
+                .andExpect(jsonPath("$").isNotEmpty())
+                .andExpect(status().isOk()).andReturn().getResponse().getContentAsString();
```

Comment:
either assert if contentString is proper uuid, or remove `.andReturn().getResponse().getContentAsString()`